pom.xml

```diff
@@ -68,6 +70,7 @@
     <metadatamappingVersion>1.3.1</metadatamappingVersion>
     <metadatasharingVersion>1.2.2</metadatasharingVersion>
     <operationtheaterVersion>1.2.1</operationtheaterVersion>
+    <openmrsERPVersion>1.0.0</openmrsERPVersion>
```

Comment:
This module is needed only when using the Billing Status widget (on the patient dashboard)
Are we using such widget with Easydoc?